tests/framework/extensions/nodes/ec2/ec2.go

```diff
@@ -184,3 +185,47 @@ func getSSHKeyName(sshKeyName string) string {
 	stringSlice := strings.Split(sshKeyName, ".")
 	return stringSlice[0]
 }
+
```

Comment:
Isn't this PR for RKE2 only changes?

@igomez06 should we separate this on two shorter PRs?. The task mention k3s/rke2 only but here's `RKE1` included.

Reply:
I agree

Reply:
Agreed